beetsplug/export.py

```diff
@@ -78,17 +101,21 @@ def commands(self):
             u'-o', u'--output',
             help=u'path for the output file. If not given, will print the data'
         )
+        cmd.parser.add_option(
+            u'-f', u'--format', default='json',
+            help=u'specify the format of the exported data. Your options are json (deafult), csv, and xml'
```

Comment:
This could be shorter: perhaps, "the output format: json (default), csv, or xml".

Reply:
Sounds good